Assets/Scripts/Puzzles/Color/UI/ColorCommandDropzone.cs

```diff
@@ -0,0 +1,45 @@
+﻿using DN.UI;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace DN.Puzzle.Color
+{
+	/// <summary>
+	/// ADD CLASS SUMMARY!
+	/// </summary>
+	public class ColorCommandDropzone : MonoBehaviour, IDroppable
+	{
+		private struct Item
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Implement 'IEquatable<T>' in value type 'Item'.](https://app.codacy.com/manual/stepperman/pvb-programming/pullRequest?prid=5403921)